### DISCUSSION TITLE
How to use textgen webui with window.ai ?

### DISCUSSION THREAD

**Discussion by syddharth** [oobabooga/text-generation-webui#1402] [Q&A]
[Window.ai](https://github.com/alexanderatallah/window.ai) is a browser extension that lets you configure AI models in one place and use them on the web.

I'd like to use the API from webui with the chrome extension.
I tried providing http://localhost:5000/api/v1/generate as the base URL for api but never got it to work. 

If someone has gotten window.ai working with a local instance (running on GPU), it'll be nice to know the workflow.

**Comment by BetaDoggo** [comment]
I think this project's api would have to be modified to accept the WindowAI requests and return results in the expected format [as shown in the readme](https://github.com/alexanderatallah/window.ai#server-api-spec).
